<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>todolist_student</class>
 <widget class="QDialog" name="todolist_student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="main_frm">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>-20</y>
     <width>1111</width>
     <height>761</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Swis721 BlkOul BT</family>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 245);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QListWidget" name="frm_nav">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>201</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius: 20px; border: 1px solid grey; 
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QListWidget" name="frm_todo">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>42</y>
      <width>531</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius: 20px; border: 1px solid grey; 
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="studtodo_table">
    <property name="geometry">
     <rect>
      <x>311</x>
      <y>126</y>
      <width>471</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius: 20px; border: 1px solid transparent; 
background-color: rgb(241, 241, 241);</string>
    </property>
    <column>
     <property name="text">
      <string>Tags:</string>
     </property>
     <property name="font">
      <font>
       <family>Dubai</family>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Task:</string>
     </property>
     <property name="font">
      <font>
       <family>Dubai</family>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Progress:</string>
     </property>
     <property name="font">
      <font>
       <family>Dubai</family>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="stud_addtodo_btn">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>640</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Display</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: rgb(21, 21, 21); border-radius: 30px; color: white;}

QPushButton:hover {border-radius: 30px; color: white; background-color: rgb(85, 170, 255);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="removepost_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>650</y>
      <width>92</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Display</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {border-radius: 20px; color: black; background-color: white; }

QPushButton:hover {border-radius: 20px; color: white; background-color: rgb(255, 85, 127);}

</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_addtodobtn">
    <property name="geometry">
     <rect>
      <x>526</x>
      <y>659</y>
      <width>83</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Display Semib</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none; border: none; 
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Add To-Do</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tags_todolist">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>130</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 BT</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none; border-radius: 18px; color: black;   border: 1px solid grey;</string>
    </property>
    <property name="text">
     <string>To-Do List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="notifyme__btn">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>69</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Display</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {border-radius: 20px; color: white; background-color: rgb(85, 170, 255);}

QPushButton:hover {border-radius: 20px; border: 1px solid #55AAFF; color: rgb(85, 170, 255); background-color: white; }</string>
    </property>
    <property name="text">
     <string>Notify Me</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_greetings">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>70</y>
      <width>124</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: NONE;</string>
    </property>
    <property name="text">
     <string>What to-do?</string>
    </property>
   </widget>
   <widget class="QFrame" name="home_frame">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>122</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius: 20px; 
background-color: rgb(221, 205, 249);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>-50</x>
       <y>-150</y>
       <width>200</width>
       <height>198</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
border-radius: 99%; /* Make it circular */
border: 2px solid purple ; /* Add a border */

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>-130</y>
       <width>200</width>
       <height>198</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
border-radius: 99%; /* Make it circular */
border: 2px solid purple ; /* Add a border */

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="mainpagetext_53">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>62</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Blk BT</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent; border: none;
color: purple;


</string>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>-56</x>
       <y>120</y>
       <width>200</width>
       <height>198</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
border-radius: 99%; /* Make it circular */
border: 2px solid purple ; /* Add a border */

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>-150</x>
       <y>-110</y>
       <width>200</width>
       <height>198</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
border-radius: 99%; /* Make it circular */
border: 2px solid purple ; /* Add a border */

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="home_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>151</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-radius: 20px; 
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="todo_btn">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>319</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius: 20px; 
background-color: rgb(198, 239, 216);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>-88</x>
       <y>120</y>
       <width>200</width>
       <height>198</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
border-radius: 99%; /* Make it circular */
border: 2px solid green ; /* Add a border */
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>-168</x>
       <y>-117</y>
       <width>200</width>
       <height>198</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
border-radius: 99%; /* Make it circular */
border: 2px solid green ; /* Add a border */

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="mainpagetext_52">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>57</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Blk BT</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent; border: none;
color: green;


</string>
     </property>
     <property name="text">
      <string>TO-DO</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-160</y>
       <width>200</width>
       <height>198</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
border-radius: 99%; /* Make it circular */
border: 2px solid green ; /* Add a border */

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="opt_frame">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>537</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius: 20px; 
background-color: rgb(211, 217, 250);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>-168</x>
       <y>-117</y>
       <width>200</width>
       <height>198</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
border-radius: 99%; /* Make it circular */
border: 2px solid blue;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="mainpagetext_61">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>56</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Blk BT</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent; border: none; 
color: blue;</string>
     </property>
     <property name="text">
      <string>OPT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>110</y>
       <width>200</width>
       <height>198</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
border-radius: 99%; /* Make it circular */
border: 2px solid blue;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>48</x>
       <y>-137</y>
       <width>200</width>
       <height>198</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
border-radius: 99%; /* Make it circular */
border: 2px solid blue;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="opt_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>146</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-radius: 20px; 
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="edittodo_btn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>69</y>
      <width>71</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Display</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {border-radius: 20px; color: white; background-color: rgb(255, 85, 127);}

QPushButton:hover {border-radius: 20px; border: 1px solid #FF557F; color: rgb(255, 85, 127); background-color: white; }
</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
